src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java

```diff
@@ -146,4 +146,12 @@ public String appendQueryStringToURL(String url, String qs) {
       return url + separator + qs;
     }
   }
+
+  public String globallyEnableGeoInstantGlobalOnPage(String pageUrl, String instantGlobal) {
+    return this.appendQueryStringToURL(pageUrl, String.format("InstantGlobals.%s=[XX]", instantGlobal));
```

Comment:
In future we could consider checking if the query param already exists in the URL and replace it's value instead of adding, but for now it's ok.